backend/src/server.ts

```diff
@@ -2,29 +2,307 @@ import express, { Express, Request, Response } from 'express';
+      },
+      {
+        "id": "22",
+        "model": "Renault Clio",
+        "brand": "Renault",
+        "pricePerDay": 55,
+        "numberOfSeats": 5,
+        "isAutomatic": true,
+        "isElectric": false,
+        "picture": "https://upload.wikimedia.org/wikipedia/commons/thumb/2/23/Renault_Clio_V_%282023%29_Esprit_Alpine_Automesse_Ludwigsburg_2023_1X7A0012.jpg/420px-Renault_Clio_V_%282023%29_Esprit_Alpine_Automesse_Ludwigsburg_2023_1X7A0012.jpg"
+      },
+      {
+        "id": "23",
+        "model": "BMW 3 Series",
+        "brand": "BMW",
+        "pricePerDay": 80,
+        "numberOfSeats": 5,
+        "isAutomatic": true,
+        "isElectric": false,
+        "picture": "https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/BMW_G20_IMG_0167.jpg/420px-BMW_G20_IMG_0167.jpg"
+      },
+      {
+        "id": "24",
+        "model": "Ford Focus",
+        "brand": "Ford",
+        "pricePerDay": 55,
+        "numberOfSeats": 5,
+        "isAutomatic": true,
+        "isElectric": false,
+        "picture": "https://upload.wikimedia.org/wikipedia/commons/thumb/7/75/2018_Ford_Focus_ST-Line_X_1.0.jpg/420px-2018_Ford_Focus_ST-Line_X_1.0.jpg"
+      },
+      {
+        "id": "25",
+        "model": "Mercedes-Benz Sprinter",
+        "brand": "Mercedes-Benz",
+        "pricePerDay": 110,
+        "numberOfSeats": 2,
+        "isAutomatic": true,
+        "isElectric": false,
+        "picture": "https://upload.wikimedia.org/wikipedia/commons/thumb/a/a2/2019_Mercedes-Benz_Sprinter_314_CDi_2.1.jpg/420px-2019_Mercedes-Benz_Sprinter_314_CDi_2.1.jpg"
+      },
+      {
+        "id": "26",
+        "model": "Ford Transit",
+        "brand": "Ford",
+        "pricePerDay": 90,
+        "numberOfSeats": 2,
+        "isAutomatic": true,
+        "isElectric": false,
+        "picture": "https://upload.wikimedia.org/wikipedia/commons/thumb/8/88/2016_Ford_Transit_350_2.2.jpg/420px-2016_Ford_Transit_350_2.2.jpg"
+      }
 ];
-
+const Odense : Location = {
+    id: '1',
+    name: 'Odense',
+    address: 'Hjallesevej 135',
+    cars: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10']
+}
+const Aarhus : Location = {
+    id: '2',
+    name: 'Aarhus',
+    address: 'Randersvej 400',
+    cars: ['11', '12', '13', '14', '15', '16', '17', '18', '19', '20']
+}
+const Copenhagen : Location = {
+    id: '3',
+    name: 'Copenhagen',
+    address: 'Amagerbrogade 100',
+    cars: ['21', '22', '23', '7', '6', '1', '2', '3', '5', '6']
+}
+const Roskilde : Location = {
+    id: '4',
+    name: 'Roskilde',
+    address: 'Ringstedvej 100',
+    cars: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10']
+}
+const locations: Location[] = [Odense, Aarhus, Copenhagen, Roskilde];
 app.get('/cars', (req: Request, res: Response) => {
     res.json(cars);
 });
+app.get('/locations', (req: Request, res: Response) => {
+    res.json(locations);
+});
+app.get('/cars/:locationName', (req: Request, res: Response) => {
```

Comment:
I believe this is a wrong use of API because it collides with a method which would be much more natural:
`/cars/:carId`
location can be a query parameter in this case
`/cars?location=Odense`
if no query parameter is given, then we return all the cars

But don't introduce this change, since I have already implemented it in next commit